models/aeif_cond_beta_multisynapse.h

```diff
@@ -392,8 +431,15 @@ class aeif_cond_beta_multisynapse : public Archiving_Node
   Buffers_ B_;
   /** @} */
 
+
+  // Access functions for UniversalDataLogger -------------------------------
+
   //! Mapping of recordables names to access functions
-  static RecordablesMap< aeif_cond_beta_multisynapse > recordablesMap_;
+  DynamicRecordablesMap< aeif_cond_beta_multisynapse > recordablesMap_;
+
+  // Utility function that inserts the synaptic conductances to the
+  // recordables map
+  inline void insert_conductance_recordables( size_t first = 0 );
```

Comment:
Remove `inline`! The functions operation is too complex to inline it; plus, modern compilers are quite good at reading our code.

Reply:
Done in eb41277